test/test_install_cmdstan.py

```diff
@@ -31,6 +38,30 @@ def test_retrieve_version(self):
         ):
             retrieve_version('no_such_version')
 
+    def test_install_cmdstan_specify_dir(self):
+        cur_cmdstan_path = ''
+        if 'CMDSTAN' in os.environ:
+            cur_cmdstan_path = os.environ['CMDSTAN']
+            del os.environ['CMDSTAN']
+        self.assertFalse('CMDSTAN' in os.environ)
+
+        version = '2.24.1'
+        tmpdir = os.path.join(DATAFILES_PATH, 'tmp-' + str(time()))
+        os.mkdir(tmpdir)
```

Comment:
`os.makedirs(tmpdir, exist_ok=True)`

Reply:
many thanks!!!

Reply:
nope, test still hangs on Windows.  without this, tests run in 12 minutes.  this test tries to do a download from GitHub - maybe not allowed in the CI setting?  in any case, I think it'll be OK to skip running this tests on Windows.